diff --git a/AdvancedTaskControl.sln b/AdvancedTaskControl.sln
index 2c43e44..7b286d9 100644
--- a/AdvancedTaskControl.sln
+++ b/AdvancedTaskControl.sln
@@ -13,6 +13,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AdvancedTaskControl.Data",
 EndProject
 Project("{E53339B2-1760-4266-BCC7-CA923CBCF16C}") = "docker-compose", "docker-compose.dcproj", "{3714F25C-FCA3-46E5-8CF6-6D8F412945B7}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AdvancedTaskControl.MessageConsumer", "src\AdvancedTaskControl.MessageConsumer\AdvancedTaskControl.MessageConsumer.csproj", "{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
@@ -71,6 +73,18 @@ Global
 		{3714F25C-FCA3-46E5-8CF6-6D8F412945B7}.Release|x64.Build.0 = Release|Any CPU
 		{3714F25C-FCA3-46E5-8CF6-6D8F412945B7}.Release|x86.ActiveCfg = Release|Any CPU
 		{3714F25C-FCA3-46E5-8CF6-6D8F412945B7}.Release|x86.Build.0 = Release|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Debug|x64.Build.0 = Debug|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Debug|x86.Build.0 = Debug|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Release|Any CPU.Build.0 = Release|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Release|x64.ActiveCfg = Release|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Release|x64.Build.0 = Release|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Release|x86.ActiveCfg = Release|Any CPU
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4}.Release|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
@@ -79,6 +93,7 @@ Global
 		{240A8ED8-8573-4DEA-B011-6BA76C4FA5C2} = {AE00BFEB-35D9-4863-AA56-BB2D4A274286}
 		{5E6D2778-9189-492F-9C33-C2C9F1A09AE0} = {AE00BFEB-35D9-4863-AA56-BB2D4A274286}
 		{8352A987-3C78-4B8C-AB5D-7E903053C433} = {AE00BFEB-35D9-4863-AA56-BB2D4A274286}
+		{C3B77870-8EAD-4CA9-874D-EC3719DDD0C4} = {AE00BFEB-35D9-4863-AA56-BB2D4A274286}
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {01F6C55C-1AD2-4C3F-8B8D-1D66CDF45F0F}
diff --git a/docker-compose.yml b/docker-compose.yml
index 782ece5..0d14c5c 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -41,6 +41,17 @@ services:
     networks:
       - pg-network
 
+  #advancedtaskcontrol.messageconsumer:
+  #  image: ${DOCKER_REGISTRY-}advancedtaskcontrolmessageconsumer
+  #  build:
+  #    context: .
+  #    dockerfile: src/AdvancedTaskControl.MessageConsumer/Dockerfile
+  #  depends_on:
+  #    - db-postgres
+  #    - rabbitmq
+  #  networks:
+  #    - pg-network
+
 volumes:
   database-data: 
   rabbitmq-data:
